include/Model/Languages.php

```diff
@@ -553,7 +562,7 @@ public function has(): bool {
 	 * }
 	 * @return array List of PLL_Language objects or PLL_Language object properties.
 	 */
-	public function get_list( $args = array() ): array {
+	public function get_list( array $args = array() ): array {
```

Comment:
Even though I like it, I wouldn't type hint this extensively used public method. This is part of our public API and I fear other developers are not as strict as us.